wrappers/odoo_client.py

```diff
@@ -35,8 +36,19 @@ def drop_attendances(self, date_window):
              ('check_out', '=', None)
              ]
         )
+        error_removing_all = False
```

Comment:
can you add a TODO? We should avoid dropping attendances if they don't change.
We could read them all 1st for the given week and delete only the ones not matching.

Reply:
Done

Reply:
where? :D